conan/tools/gnu/pkgconfigdeps.py

```diff
@@ -346,8 +363,8 @@ def _update_pc_files(info):
         # Issue related: https://github.com/conan-io/conan/issues/10341
         pkg_name = _get_package_name(self._dep, self._build_context_suffix)
         if f"{pkg_name}.pc" not in pc_files:
-            package_info = _PCInfo(pkg_name, pkg_requires, f"Conan package: {pkg_name}", None,
-                                   _get_package_aliases(self._dep))
+            package_info = _PCInfo(pkg_name, pkg_requires, f"Conan package: {pkg_name}",
```

Comment:
This change not really necessary, the ``shortened_content`` still do not require the ``cpp_info``

Reply:
it needs it for `custom_content`...

Reply:
I see, yes, you are right I got confused looking at my local copy of the code.